src/Button/Button.test.tsx

```diff
@@ -44,11 +44,12 @@ describe("Component: Button", () => {
     describe("Should render supported themes", () => {
         const list: Array<ButtonTestItem<ButtonTheme, string>> = [
             { value: "primary", expected: "btn-primary" },
-            { value: "secondary", expected: "btn-outline-primary" },
-            { value: "alternative", expected: "btn-secondary" },
+            { value: "outline-primary", expected: "btn-outline-primary" },
+            { value: "secondary", expected: "btn-secondary" },
             { value: "ghost-dark", expected: "btn-ghost-dark" },
             { value: "ghost-light", expected: "btn-ghost-light" },
-            { value: "anchor", expected: "btn-anchor" },
+            { value: "anchor", expected: "btn-link" },
```

Comment:
i can't see anchor anymore in the bootstrap/buttons, maybe we should remove it from here as well ?

Reply:
We should only keep it so that we don't break the styles of people who are already using it. We should mention it in the release notes that it is deprecated and will be removed soon. Give them a change to switch to `link` before we remove it.

Reply:
Yes understand, can we add @deprecated to it so that we keep track of it?